packages/javascript-sdk/src/fr-webauthn/index.ts

```diff
@@ -162,8 +176,7 @@ abstract class FRWebAuthn {
         hiddenCallback.setInputValue(`${WebAuthnOutcome.Error}::${error.name}:${error.message}`);
         throw error;
       }
-
-      hiddenCallback.setInputValue(outcome);
+      hiddenCallback.setInputValue(deviceName ? `${outcome}::${deviceName}` : outcome);
```

Comment:
hypothetically, we could give `hiddenCallback` a generic that defaults to `unknown` and then we can provide that here for more type inference and passing.